R/returnData.R

```diff
@@ -101,7 +107,7 @@ getSodaTypes <- function(response) {
 #' @importFrom mime guess_type
 #' 
 #' @export
-read.socrata <- function(url, app_token = NULL) {
+read.socrata <- function(url, app_token = NULL, email = NULL, password = NULL) {
```

Comment:
are you sure this comes from the newest dev branch @tomschenkjr @marks  ? 
https://github.com/Chicago/RSocrata/blob/dev/R/returnData.R

Rebase maybe ?